### ISSUE TITLE
Auto Win

### ISSUE THREAD

**Issue by yGleam** [music-discussion/TownOfHost-TheOtherRoles#67]
I've been spending all day activating and reactivating roles to see which one is causing the problem. I cant find the issue as soon as I press start it goes to the win screen even with a full lobby of players. also the opportunist isn't even in the game anymore.

**Comment by music-discussion** [comment]
It’s been replaced by Survivor. A more better role as you don’t just have to hide all game. Roles such as Exe can still turn into Opportunist.

**Comment by SkullCreeper** [comment]
Double check your Neutral Max and Min settings, neutral roles, gamemode settings, **percentage.txt**, and crewmate roles.

**Comment by yGleam** [comment]
as far as I remember I have them at 0 on everyone I can show a screenshot of my current roles if you'd like.

**Comment by yGleam** [comment]
![image](https://user-images.githubusercontent.com/96977336/189784775-9398a8c5-916d-4c4c-b86a-610248e06c34.png)

**Comment by yGleam** [comment]
![image](https://user-images.githubusercontent.com/96977336/189784843-4dff1ef3-5f27-4a6d-97ba-d04696e0b56f.png)

**Comment by yGleam** [comment]
also I have the actual percentages set to 100% on every activated role

**Comment by Tommy-XL** [comment]
If the game still ends, try disabling this setting
![unknown-46](https://user-images.githubusercontent.com/104814436/189785428-d464785b-e34e-4a05-8f25-923e3449ec04.png)

**Comment by yGleam** [comment]
I just did that and started it and I see this still..
![image](https://user-images.githubusercontent.com/96977336/189785431-e9d8a69b-4e12-46a1-8042-42d92297961b.png)

**Comment by yGleam** [comment]
oh ok ill try that.

**Comment by yGleam** [comment]
still is happening 
![image](https://user-images.githubusercontent.com/96977336/189785638-dc570c7f-02e0-4b23-8f9b-d4bbff163af7.png)

**Comment by yGleam** [comment]
I believe at this point its one of the roles I have on because when I take off every role and play normally it works great.

**Comment by yGleam** [comment]
ok ok I found the problem! I removed the Medium role and turned on the Imposters Know Egoist setting it works now!

**Comment by Tommy-XL** [comment]
Hmm, try to experiment with roles, remove some neutral roles like the Executioner

**Comment by yGleam** [comment]
ok yup I just enabled the Medium role and the problem occurred

**Comment by Tommy-XL** [comment]
Hm, Okey I will remember that the role of Medium causes problems

**Comment by yGleam** [comment]
whyyy its back. I cant play I feel bad for all the people who join just to get teased by the screen ending.

**Comment by Tommy-XL** [comment]
Have you changed the game settings or left everything as it is?

**Comment by yGleam** [comment]
I think I added and removed a role I cant remember which one

**Comment by yGleam** [comment]
well this is so weird but i turned off tons of settings like set the min max neutral killing settings back to 0 and it fixed it

**Comment by yGleam** [comment]
ok to dumb it down if I enable any settings it breaks it

**Comment by yGleam** [comment]
and if I enable/disable them it fixes it almost like random number generation

**Comment by yGleam** [comment]
as if its based on luck or chance

**Comment by yGleam** [comment]
wow. i just started it and it auto ended then i just tried starting again right after and it worked its so buggy

**Comment by yGleam** [comment]
Theres no like set in stone fix its so random if its works or not

**Comment by larushco** [comment]
this is helpful, thank you

**Comment by yGleam** [comment]
I give up. I hope a new release comes soon I miss playing this.

**Comment by larushco** [comment]
is it the 0.8.3.1b

**Comment by yGleam** [comment]
Yes, its pretty buggy for a lot of people. so I would just resort to older versions they may not be polished and have their own issues. but at least you can load a game.

**Comment by iChlorox** [comment]
![image](https://user-images.githubusercontent.com/91646808/190042982-39303f23-b0ee-4d0b-9023-4c346c617db6.png)
![Uploading image.png…]()
It really doesn't! i have been playing with Medium enabled since the release!

**Comment by iChlorox** [comment]
![image](https://user-images.githubusercontent.com/91646808/190043053-32568720-3b2e-4c9d-9967-9f203039683e.png)
2nd image was still uploading when i sent it

**Comment by SkullCreeper** [comment]
Yeah, can confirm Medium is not the issue.

**Comment by yGleam** [comment]
Well I cant just act like I'm playing I'm gonna say when somethings wrong I literally spent hours activating certain settings and testing stuff and I'm stuck I've been forced to play older versions :/

**Comment by yGleam** [comment]
I guess good for you? idk, I'm tired of this buggy version.

**Comment by yGleam** [comment]
what are your settings so I can compare mine.

**Comment by yGleam** [comment]
fixed it for real this time.

Roles look Similar to ToU  OFF

 Min & Max Neutral Killing Settings all on 0
 
 Imposters Know Egoist ON
 
 and thats working for me.

**Comment by yGleam** [comment]
It was working by myself but as soon as I got a lobby of 15/15 it auto wins once again..

**Comment by yGleam** [comment]
When you can show your settings please I cant play a single game without it going to crewmate win screen

**Comment by iChlorox** [comment]
Alright i'll share my settings with you and you can replace imp for imp, crew for crew ,nnk for nnk, nk for nk, support for support
![image](https://user-images.githubusercontent.com/91646808/190123096-9a87de98-4676-4995-8603-dc2e320d9424.png)
![image](https://user-images.githubusercontent.com/91646808/190123118-fbde7c6b-6b79-43de-a4b4-23cde3cb1903.png)
![image](https://user-images.githubusercontent.com/91646808/190123145-29e10503-88bc-4a10-b67e-33bb5e176941.png)
![image](https://user-images.githubusercontent.com/91646808/190123176-cb9ab2ed-3943-4330-abec-8f8ae89a9186.png)
![image](https://user-images.githubusercontent.com/91646808/190123205-b4b8d02d-a950-45db-a813-8a155ec34bf6.png)
![image](https://user-images.githubusercontent.com/91646808/190123241-33c31660-decb-48cc-a85a-779f87e39487.png)
![image](https://user-images.githubusercontent.com/91646808/190123279-11fa916a-0bfe-4f2e-9420-99dfb8241e5a.png)
![image](https://user-images.githubusercontent.com/91646808/190123309-fccad820-65b7-442b-abfa-d517f13c5d27.png)
Percentages are.. the ones written down i use laptop percentages (from next update, but the roles are the same as the ones i was using in 0.8.3.1b
![image](https://user-images.githubusercontent.com/91646808/190123714-748a2321-e6c8-4618-bf6b-13c347042286.png)

**Comment by yGleam** [comment]
thanks for taking the time to send this 💯

**Comment by yGleam** [comment]
if you're on a even more recent build than 0.8.3.1b then that's why yours is working and mine isn't because im using 0.8.3.1b

**Comment by yGleam** [comment]
i copied pretty much everything u told me

**Comment by yGleam** [comment]
![image](https://user-images.githubusercontent.com/96977336/190134189-b2689235-f6df-449c-81b9-86349d5651e4.png)
![image](https://user-images.githubusercontent.com/96977336/190134220-cda07958-3c8b-4761-99bf-2a050029e5e8.png)
![image](https://user-images.githubusercontent.com/96977336/190134260-8b523119-7cd7-448b-a708-6f741b64067b.png)
![image](https://user-images.githubusercontent.com/96977336/190134287-da694b5f-7ff2-4bb4-b23c-f75a8dc13bf3.png)
![image](https://user-images.githubusercontent.com/96977336/190134371-f11fd86b-84cd-4d6f-9dad-e4c5f8e4a648.png)
![image](https://user-images.githubusercontent.com/96977336/190134406-ab93d1f3-0cf3-48d4-8c74-6fdd377c695b.png)
![image](https://user-images.githubusercontent.com/96977336/190134503-9226505a-7bdb-493e-a887-73cb874a3f47.png)
![image](https://user-images.githubusercontent.com/96977336/190134521-d2c735e7-9249-4ca3-981b-82e254101cfc.png)
![image](https://user-images.githubusercontent.com/96977336/190134569-7bf26c84-e185-4057-a709-1177f96f5c6f.png)

**Comment by yGleam** [comment]
![image](https://user-images.githubusercontent.com/96977336/190134671-2827fc40-25e6-487b-881f-c02a1634e6db.png)

**Comment by iChlorox** [comment]
Yeah the works should run in that version normally

**Comment by yGleam** [comment]
what do you mean the works

**Comment by yGleam** [comment]
I'm sorry I just am desperate to play this amazing mod again😅

**Comment by yGleam** [comment]
Is there a way I can get a even more recent dev build?

**Comment by iChlorox** [comment]
Yeah build the github source, but in the setting set you have, if you change the neutural killings min max from 1-2 to 1-3 or 3-3 you will be able to run it with 12+ people in the lobby

**Comment by yGleam** [comment]
will this work?
![image](https://user-images.githubusercontent.com/96977336/190136551-edad6bd1-4a2d-4e0e-95a2-c0e5f4d53342.png)

**Comment by yGleam** [comment]
how can I download the more recent dev beta builds?

**Comment by yGleam** [comment]
its still happening!
![image](https://user-images.githubusercontent.com/96977336/190137042-808460e7-44eb-4362-b41f-d700da0d266e.png)

**Comment by yGleam** [comment]
I wish i can download a dev build

**Comment by BornBasic** [comment]
yGleam do u have discord? my settings work for full lobby

**Comment by yGleam** [comment]
yes Gleam#4703

**Comment by yGleam** [comment]
not sure what that means

**Comment by music-discussion** [comment]
that will only happen if the max is lower than the min.

**Comment by Tommy-XL** [comment]
Auto win crew will be fixed in the next version

**Comment by MrDonkus** [comment]
> Auto win crew will be fixed in the next version

Can confirm from testing it tonight

**Comment by Tommy-XL** [comment]
Yea
